packaging/linux/deb/package_binaries.sh

```diff
@@ -81,6 +81,14 @@ build_one_architecture() {
     > "$cron_file"
   chmod 755 "$cron_file"
 
+  # Whitelist for NativeMessaging
+  kbnm_bin="/usr/bin/kbnm"
+  kbnm_file="$dest/etc/opt/chrome/native-messaging-hosts/io.keybase.kbnm"
+  mkdir -p "$(dirname "$kbnm_file")"
+  cat "$here/host_json.template" \
+    | sed "s|@@HOST_PATH@@|/usr/local/bin/$kbnm_bin|g"
+  chmod 644 "$kbnm_file"
```

Comment:
I think we can skip this line, if we expect it to be a no-op? (I don't have strong opinions about the True and Righteous Way to respect the umask though.)